index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title> Portfolio </title>
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+ 
+ <div class="container">
+  
+  <div class="top-icons">
+   <div class="left-top"> <span style="font-family: Roboto; font-weight: 500;">9:41</span></div>
+   <div class="right-top">
+   <img src="/images/Bars/Status Bars/Cellular Connection.png">
+   <img src="/images/Bars/Status Bars/Wifi.png">
+   <img src="/images/Bars/Status Bars/Battery.png">
+  </div>
```

Comment:
- [ ] I think you should remove this part of your document. This is only added to the figma design to show the representation of a mobile phone.

Reply:
got it ! thanks